pep-0680.rst

```diff
@@ -2,13 +2,13 @@ PEP: 680
 Title: tomllib: Support for parsing TOML in the Standard Library
 Author: Taneli Hukkinen, Shantanu Jain <hauntsaninja at gmail.com>
```

Comment:
```suggestion
Author: Taneli Hukkinen, Shantanu Jain <hauntsaninja at gmail.com>
```

Would be good to include @hukkin 's email...

Reply:
I can if necessary, but also not a huge fan of sharing any of my email addresses publicly on the Internet. Is there a good reason for having it, or a policy requiring it?

Reply:
Actually, email address is explicitly stated to be optional optional, not recommended or required as I thought, [per PEP 1](https://www.python.org/dev/peps/pep-0001/#pep-header-preamble). My mistake; sorry about that!